components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/docs/examples/apis/search_get.yaml

```diff
@@ -0,0 +1,86 @@
+- id: searchApis
```

Comment:
Shall we add a title. Eg: https://github.com/wso2/carbon-apimgt/blob/85783b1994d9b3c9c9cf081c627f67cacc3771de/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/docs/examples/apis/apis_copyapi.yaml#L3
**Suggestion:**
title: Search APIs and Documentation By a Given Query